src/cmd/cli/command/compose.go

```diff
@@ -89,6 +90,13 @@ func makeComposeUpCmd() *cobra.Command {
 			tailCtx, cancelTail := context.WithCancelCause(cmd.Context())
 			defer cancelTail(nil) // to cancel WaitServiceState and clean-up context
 
+			ctx := tailCtx
```

Comment:
wonder why you'd rename `tailCtx`.It's fine by me, but then fix the occurrence in the comments too, line 173.